cc/defs.bzl

```diff
@@ -215,6 +223,18 @@ def swift_cc_test(name, type, **kwargs):
 
     _ = kwargs.pop("nocopts", [])  # To handle API compatibility.
 
+    srcs_name = name + "_srcs"
+    srcs = kwargs.get("srcs", [])
+
+    native.filegroup(
```

Comment:
I think it might be a little confusing to users or non-obvious that we generate an additional target for the sources.

Some docstring like I've provided in the `swift_cc` macros would go a long way here.

See this project for how to add documentation to rules: https://github.com/bazelbuild/stardoc

Reply:
Added